.github/actions/test_deploy_win.ps1

```diff
@@ -0,0 +1,251 @@
+$VERSION = Get-Content src/VERSION
+[version]$VERSION = $VERSION -replace '[v]',''
+$MAJOR=$VERSION.Major
+$MINOR=$VERSION.Minor
+$SHA= git rev-parse --short $args[0]
+
+
+$WAZUH_MANAGER="1.1.1.1"
+$WAZUH_MANAGER_PORT="7777"
+$WAZUH_PROTOCOL="udp"
+$WAZUH_REGISTRATION_SERVER="2.2.2.2"
+$WAZUH_REGISTRATION_PORT="8888"
+$WAZUH_REGISTRATION_PASSWORD="password"
+$WAZUH_KEEP_ALIVE_INTERVAL="10"
+$WAZUH_TIME_RECONNECT="10"
+$WAZUH_REGISTRATION_CA="/var/ossec/etc/testsslmanager.cert"
+$WAZUH_REGISTRATION_CERTIFICATE="/var/ossec/etc/testsslmanager.cert"
+$WAZUH_REGISTRATION_KEY="/var/ossec/etc/testsslmanager.key"
+$WAZUH_AGENT_NAME="test-agent"
+$WAZUH_AGENT_GROUP="test-group"
+$ENROLLMENT_DELAY="10"
+
+function install_wazuh($vars)
+{
+    Write-Output "Testing the following variables $vars"
+    Start-Process  C:\Windows\System32\msiexec.exe -ArgumentList  "/i wazuh-agent-$VERSION-0.commit$SHA.msi /qn $vars" -wait
+}
+
+function remove_wazuh
+{
+    Start-Process  C:\Windows\System32\msiexec.exe -ArgumentList "/x wazuh-agent-$VERSION-commit$SHA.msi /qn" -wait
+}
+
+function test($vars)
```

Comment:
I see too much-repeated code. Please implement a template function (or functions) that avoids the code repeating.